roles/ocp4-cloud-ipi/templates/private_ssh_key.pub.j2

```diff
@@ -0,0 +1 @@
+{{ private_ssh_key_pub }}
```

Comment:
should we just call this public_ssh_key ? I think it is less confusing (same for filename)

Reply:
yes

Reply:
done